docker/docker-compose.yaml

```diff
@@ -22,9 +22,9 @@ services:
       - CAP_NET_RAW
       - CAP_NET_BIND_SERVICE
 
-  sanity:
```

Comment:
As with NIO, I believe this change requires coordination with @tomerd in order to ensure the CI continues to function effectively.